scrapers/koji.py

```diff
@@ -113,6 +113,70 @@ def get_build_tags(self, build_id):
 
         return self.teiid.query(sql=sql_query)
 
+    def get_tag_info(self, tag_name):
+        """
+        Query Teiid for tag_id of a tag and build_ids associated to it.
+
+        :param str tag_name: tag name
+        :return: a list of dictionaries
+        :rtype: list
+        """
+        sql_query = """
+            SELECT tags.id, tag_listing.build_id
+            FROM brew.tag_listing AS tag_listing
+            LEFT JOIN brew.tag AS tags ON tag_listing.tag_id = tags.id
+            WHERE tag_listing.active = True AND tags.name = '{}';
+            """.format(tag_name)
+
+        return self.teiid.query(sql=sql_query)
+
+    def get_build_info(self, build_id):
+        """
+        Query Teiid for build info.
+
+        :param int build_id: the Koji build's ID
+        :return: a list of dictionaries
+        :rtype: list
+        """
+        sql_query = """
+                SELECT
+                events.time as creation_time,
+                build.completion_time,
+                build.epoch,
+                build.extra,
+                build.id,
+                brew.users.name as owner_name,
+                brew.users.krb_principal as owner_username,
+                package.name as package_name,
+                build.release,
+                build.start_time,
+                build.state,
+                build.task_id,
+                build.version
+            FROM build
+            LEFT JOIN events ON build.create_event = events.id
+            LEFT JOIN package ON build.pkg_id = package.id
+            LEFT JOIN brew.users ON build.owner = brew.users.id
+            WHERE build.id = {};
+        """.format(build_id)
+
+        return self.teiid.query(sql=sql_query)
+
+    def process_tag_builds(self, tag_builds_info):
+        """
+        Update Neo4j with builds attached to a tag.
+
+        :param dict tag_builds_info: build id's associated to a tag
+        """
+        builds = []
+        for item in tag_builds_info:
+            build = self.get_build_info(item['build_id'])
+            if not build:
+                continue
```

Comment:
Maybe add a `log.warn` here?

Reply:
This concern goes away since we are using a consolidated SQL query and we wouldn't know if a build is missing and isn't returned